### PR TITLE
fix(suite-native): broken layout on android

### PR DESCRIPTION
**Pull request by vytick**
old: 
<span>  

new:
</span>

### CODE REVIEW

File: suite-native/module-connect-device/src/screens/ConnectingDeviceScreen.tsx [hunk 1]
```diff
@@ -28,12 +28,19 @@ export const ConnectingDeviceScreen = () => {
         <ConnectDeviceSreenView style={applyStyle(screenStyle)}>
             <VStack spacing="medium" alignItems="center">
                 <ActivityIndicator size="large" />
-                <Text variant="titleMedium">
-                    <Translation id="moduleConnectDevice.connectingDeviceScreen.title" />{' '}
+                <Box
+                    style={{
```
Comment (juriczech) [inline suite-native/module-connect-device/src/screens/ConnectingDeviceScreen.tsx]: You can pass these as props without using the style prop

**Comment by vytick** [comment]
Ah my fault, I planned to rewrite this once it layouts corrctly and forgot
afterwards. Good spot!

File: suite-native/module-connect-device/src/screens/ConnectingDeviceScreen.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/module-connect-device/src/screens/ConnectingDeviceScreen.tsx]: fixed in 450f5c103b

**Review by juriczech — APPROVED** [review]

**Pull request merged**